### PR TITLE
test: slashing todos

### PR DESCRIPTION
**Pull request by 0xClandestine**

### CODE REVIEW

File: src/test/unit/AllocationManagerUnit.t.sol [hunk 1]
```diff
@@ -996,115 +1007,135 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
...
+        _registerForOperatorSet(defaultOperator, operatorSet2);
+
+        cheats.prank(defaultOperator);
+        allocationManager.modifyAllocations(allocateParams);
+        cheats.roll(block.number + DEFAULT_OPERATOR_ALLOCATION_DELAY);
+
+        // Get slashable shares for each operatorSet
+        address[] memory operatorArray = new address[](1);
+        operatorArray[0] = defaultOperator;
+        // (, uint256[][] memory slashableSharesOpset1_preSlash) = allocationManager
```
Comment (ypatil12) [inline src/test/unit/AllocationManagerUnit.t.sol:1064]: Any reason why this is commented?

File: src/test/unit/AllocationManagerUnit.t.sol [hunk 2]
```diff
@@ -996,115 +1007,135 @@ contract AllocationManagerUnitTests_SlashOperator is AllocationManagerUnitTests
...
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 1), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+        // (, uint256[][] memory slashableSharesOpset2_postSlash) = allocationManager
+        //     .getMinDelegatedAndSlashableOperatorSharesBefore(
+        //     OperatorSet(defaultAVS, 2), operatorArray, defaultStrategies, uint32(block.number + 1)
+        // );
+
+        // assertEq(20e18, slashableSharesOpset1_postSlash[0][0], "slashableShares of opSet_1 should be 20e18");
+        // assertEq(
```
Comment (ypatil12) [inline src/test/unit/AllocationManagerUnit.t.sol:1111]: Are we testing this in a separate func?

**Review by ypatil12 — APPROVED** [review]
Approving, refactor is clean.

File: src/test/unit/AllocationManagerUnit.t.sol (hunk 1 above)
Comment (0xClandestine) [reply src/test/unit/AllocationManagerUnit.t.sol:1064]: unused var

File: src/test/unit/AllocationManagerUnit.t.sol (hunk 2 above)
Comment (0xClandestine) [reply src/test/unit/AllocationManagerUnit.t.sol:1111]: The fn was removed, need to find where this state lives now.

File: src/test/unit/AllocationManagerUnit.t.sol (hunk 1 above)
Comment (0xClandestine) [reply src/test/unit/AllocationManagerUnit.t.sol:1064]: wait no, same as other

**Pull request merged**